documentation/principles/cost-effective.md

```diff
@@ -2,12 +2,17 @@
 layout: sub-navigation
 order: 105
 title: Cost Effective
-excerpt: Short extract
+excerpt: We aim to help teams reduce build and run costs.
 tags:
 - principle
 eleventyNavigation:
   parent: principles
 ---
-Core Cloud operates consumption-based billing meaning teams only pay for their usage.\
-This incentivises teams to take responsibility and optimise resources attributed to them.\
-Core Cloud supports teams in optimising their resources using techniques like autoscaling, downscaling, spot pricing and ARM powered instances.
\ No newline at end of file
+**We operate consumption-based billing**. \
+This means teams only pay for their usage and are incentivised to optimise their resources. 
+
+**We support teams in optimising their resources** using techniques like autoscaling, downscaling, spot pricing and ARM powered instances.
+
+**Sensible defaults** in our service catalogue mean teams can run cheaply by default and won't have to spend development effort reinventing the wheel.
+
+**Shared capability** of common tooling reduces the cost and effort for each team.
```

Comment:
```suggestion
**Shared services** will be used to reduce the cost and effort for each customer wherever we can maintain a customer's isolation.
```
I think shared resources goes beyond tooling and I'm not so comfortable with the term `tooling` yet as it seems to get everyone talking about common pipeline and CTF etc.

Reply:
Can we meet in the middle and drop `wherever we can maintain a customer's isolation` as too involved in implementation specifics?

Reply:
Sure, now I read it back I wonder why I tacked that onto the end. I know the isolation/insulation is important but it's different than the cost principles and so I shouldn't tack onto the end of everything